draft-iab-protocol-maintenance.md

```diff
@@ -179,9 +178,9 @@ for bug compatible".
 
 For example, in TLS {{?TLS=RFC8446}}, extensions use a tag-length-value format
 and they can be added to messages in any order.  However, some server
-implementations terminate connections if they encounter a TLS ClientHello
+implementations terminated connections if they encounter a TLS ClientHello
```

Comment:
I'll you decide since you're the native speaker but shouldn't the two tenses agree?
```suggestion
implementations terminated connections if they encountered a TLS ClientHello
```